x/registry/client/cli/tx_register_top_level_domain.go

```diff
@@ -0,0 +1,40 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/mycel-domain/mycel/x/registry/types"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdRegisterTopLevelDomain() *cobra.Command {
```

Comment:
Add missing argument `name` and `registrationPeriodInYear`

Reply:
added